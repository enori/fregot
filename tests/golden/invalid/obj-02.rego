package fregot.tests.invalid.obj_02

inconsistent = {
  "k": "hi",
  "k": "bye",
}
