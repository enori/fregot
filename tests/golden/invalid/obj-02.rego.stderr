fregot (compile error):
  "obj-02.rego" (line 5, column 3):
  Duplicate object key "k":

    5|   "k": "bye",
         ^^^

  "obj-02.rego" (line 4, column 3):
  Duplicate object key "k":

    4|   "k": "hi",
         ^^^
